diff --git a/extras/com.giadamusic.Giada.desktop b/extras/com.giadamusic.Giada.desktop
index 3c815800..6b325958 100644
--- a/extras/com.giadamusic.Giada.desktop
+++ b/extras/com.giadamusic.Giada.desktop
@@ -3,7 +3,6 @@ Version=1.0
 Type=Application
 Name=Giada
 GenericName=Drum machine and loop sequencer
-Comment=Drum machine and loop sequencer
 GenericName[de]=Drum Maschine und Loop Sequenzer
 GenericName[es]=Caja de ritmos y secuenciador de loops
 GenericName[fr]=Boîte à rythme et séquenceur de boucle
