diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ee78947..821a04e8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -660,7 +660,7 @@ if(DEFINED OS_LINUX)
 	list(APPEND LIBRARIES
 		${X11_LIBRARIES} ${X11_Xrender_LIB} ${X11_Xft_LIB} ${X11_Xfixes_LIB}
 		${X11_Xinerama_LIB} ${X11_Xcursor_LIB} ${X11_Xpm_LIB} ${LIBRARY_FONTCONFIG}
-		${JACK_LDFLAGS} ${CMAKE_DL_LIBS} pthread stdc++fs)
+		${JACK_LDFLAGS} ${CMAKE_DL_LIBS} pthread)
 
 	if (WITH_ALSA)
 		find_package(ALSA REQUIRED)
