diff --git a/CMakeLists.txt b/CMakeLists.txt
index 06d45aa9..e812d4a5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -437,7 +437,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
 		${X11_LIBRARIES} ${X11_Xrender_LIB} ${X11_Xft_LIB} ${X11_Xfixes_LIB}
 		${X11_Xinerama_LIB} ${X11_Xcursor_LIB} ${X11_Xpm_LIB} ${LIBRARY_PULSE} 
 		${LIBRARY_PULSE_SIMPLE} ${LIBRARY_FONTCONFIG} ${LIBRARY_JACK}
-		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread stdc++fs)
+		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread)
 
 	list(APPEND PREPROCESSOR_DEFS
 		__LINUX_ALSA__
@@ -573,4 +573,4 @@ target_sources(giada PRIVATE ${SOURCES})
 target_compile_definitions(giada PRIVATE ${PREPROCESSOR_DEFS})
 target_include_directories(giada PRIVATE ${INCLUDE_DIRS})
 target_link_libraries(giada PRIVATE ${LIBRARIES})
-target_compile_options(giada PRIVATE ${COMPILER_OPTIONS})
\ No newline at end of file
+target_compile_options(giada PRIVATE ${COMPILER_OPTIONS})
