diff --git a/CMakeLists.txt b/CMakeLists.txt
index c1571c58..8399d86e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -395,7 +395,7 @@ if(DEFINED OS_LINUX)
 		${X11_LIBRARIES} ${X11_Xrender_LIB} ${X11_Xft_LIB} ${X11_Xfixes_LIB}
 		${X11_Xinerama_LIB} ${X11_Xcursor_LIB} ${X11_Xpm_LIB} ${LIBRARY_PULSE}
 		${LIBRARY_PULSE_SIMPLE} ${LIBRARY_FONTCONFIG} ${JACK_LDFLAGS}
-		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread stdc++fs)
+		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread)
 
 	if (WITH_ALSA)
 		list(APPEND PREPROCESSOR_DEFS __LINUX_ALSA__)
@@ -571,4 +571,4 @@ elseif(DEFINED OS_MACOS)
 	set_target_properties(giada PROPERTIES
 		XCODE_ATTRIBUTE_ENABLE_HARDENED_RUNTIME YES)
 
-endif()
\ No newline at end of file
+endif()
