diff --git a/CMakeLists.txt b/CMakeLists.txt
index c58d8d14..cfd418e6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -291,7 +291,7 @@ endif()
 
 set(FLTK_SKIP_FLUID TRUE)  # Don't search for FLTK's fluid
 set(FLTK_SKIP_OPENGL TRUE) # Don't search for FLTK's OpenGL
-find_package(FLTK CONFIG REQUIRED)
+find_package(FLTK REQUIRED)
 list(APPEND LIBRARIES fltk fltk_gl fltk_forms fltk_images)
 message("FLTK library found in " ${FLTK_DIR})
 
@@ -387,7 +387,7 @@ if(DEFINED OS_LINUX)
 		${X11_LIBRARIES} ${X11_Xrender_LIB} ${X11_Xft_LIB} ${X11_Xfixes_LIB}
 		${X11_Xinerama_LIB} ${X11_Xcursor_LIB} ${X11_Xpm_LIB} ${LIBRARY_PULSE}
 		${LIBRARY_PULSE_SIMPLE} ${LIBRARY_FONTCONFIG} ${LIBRARY_JACK}
-		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread stdc++fs)
+		${CMAKE_DL_LIBS} ${ALSA_LIBRARIES} pthread)
 
 	list(APPEND PREPROCESSOR_DEFS
 		WITH_AUDIO_JACK
@@ -529,7 +529,7 @@ target_compile_options(giada PRIVATE ${COMPILER_OPTIONS})
 # Install rules
 # ------------------------------------------------------------------------------
 
-install(TARGETS giada DESTINATION ${CMAKE_INSTALL_PREFIX})
+install(TARGETS giada DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
 # ------------------------------------------------------------------------------
 # Extra
diff --git a/src/gui/dialogs/browser/browserBase.h b/src/gui/dialogs/browser/browserBase.h
index c0a79af5..4781d732 100644
--- a/src/gui/dialogs/browser/browserBase.h
+++ b/src/gui/dialogs/browser/browserBase.h
@@ -30,6 +30,7 @@
 #include "core/types.h"
 #include "gui/dialogs/window.h"
 #include <functional>
+#include <string>
 
 class Fl_Group;
 class geCheck;
